frontend/src/pages/details-page.tsx

```diff
@@ -56,7 +56,7 @@ export default function DetailsPage() {
         </div>
         <div className="mx-auto flex max-w-4xl flex-col items-center gap-y-4 px-4 py-10">
           <div>
-            <p className="leading-7 text-light-secondary dark:text-dark-secondary md:text-lg">
+            <p className="leading-7 text-light-secondary dark:text-dark-secondary md:text-lg mb-[1rem]">
               {post.description}
```

Comment:
Please remove hard code, and why are we adding this bottom margin? though we have added I see in desktop screen the UI is not as per desired code.
<img width="1637" alt="image" src="https://github.com/krishnaacharyaa/wanderlust/assets/116620586/536af87a-7189-45db-a437-1ee4d0f600e7">

Reply:
Done!